LoopKitUI/Views/Information Screens/DeliveryLimitsInformationView.swift

```diff
@@ -47,7 +47,7 @@ public struct DeliveryLimitsInformationView: View {
             Text(DeliveryLimits.Setting.maximumBasalRate.title)
             .font(.headline)
             VStack(alignment: .leading, spacing: 20) {
-                Text(String(format: LocalizedString("Maximum Basal Rate is the maximum automatically adjusted basal rate that %1$@ is allowed to enact to help reach your correction range.", comment: "Information about maximum basal rate (1: app name)"), appName))
+                Text(LocalizedString("Maximum Basal Rate is the maximum automatically adjusted basal rate that \(appName) is allowed to enact to help reach your correction range.", comment: "Information about maximum basal rate"))
```

Comment:
This needs to be a format string. Simple string interpolation doesn't work for localization.